fms_core/admin.py

```diff
@@ -73,6 +73,16 @@ class SampleAdmin(AggregatedAdmin):
         ("Additional Information", {"fields": ("experimental_group", "tissue_source", "phenotype", "comment")}),
     )
 
+    # def individual_participant_id(self, obj):
```

Comment:
this is according to the docs FK related objects can be added to list_display [docs](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#list-display)

but it fails

Reply:
what are you trying to do with this field?